.github/workflows/ci.yml

```diff
@@ -140,5 +139,3 @@ jobs:
           rm -rf dts-release-cicd-pipeline
           rm -f ~/.ssh/gitea-key
           rm -rf build
-          rm -f ~/.ssh/config
```

Comment:
You should add `rm -f ~/.ssh/config_deploy || true`.
Or change `config_deploy` creation earlier from `>>` to `>` as right now each time you run worker it will append the same thing to `config_deploy`

Reply:
https://github.com/Dasharo/meta-dts/pull/146/commits/38899df8e2092ecf8499a9371faaedfb78fb3f72#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR64